assets/helpers/subscriptions.js

```diff
@@ -25,8 +26,7 @@ export type ComponentAbTest = {
   variant: string,
 };
 
-
-// ----- Config ----- //
+export type DigitalBillingPeriod = 'monthly' | 'yearly';
```

Comment:
Could this be `'month' | 'year'` to fit in with the generic `BillingPeriod` type?